ledger/controllers/wallet.js

```diff
@@ -468,12 +468,199 @@ v2.lookup = { handler: (runtime) => {
   }
 }
 
+/*
+   GET /v1/wallet/stats
+ */
+
+v1.getStats =
+{ handler: (runtime) => {
+  return async (request, reply) => {
+    const debug = braveHapi.debug(module, request)
+    const wallets = runtime.database.get('wallets', debug)
+
+    let values = await wallets.aggregate([{
+      $match: {
+        paymentId: {
+          $nin: ['', null]
+        }
+      }
+    }, {
+      $project: {
+        _id: 0,
+        walletProviderBalance: '$balances.balance',
+        created: {
+          $dateToString: {
+            format: '%Y-%m-%d',
+            date: '$_id'
+          }
+        },
+        contributed: {
+          $cond: {
+            if: {
+              $gt: ['$paymentStamp', 0]
+            },
+            then: 1,
+            else: 0
+          }
+        },
+        grants: {
+          $ifNull: ['$grants', []]
```

Comment:
I think we care mostly about active grants, we may want to use the logic below checking status here